core/src/config_manager.cpp

```diff
@@ -301,6 +301,11 @@ QString ConfigManager::serverName()
     return m_settings->value("Options/server_name", "An Unnamed Server").toString();
 }
 
+QString ConfigManager::serverNick()
+{
+    return m_settings->value("Options/server_nick", "").toString();
```

Comment:
```suggestion
    return m_settings->value("Options/server_nick", "NickName").toString();
``` 
If the key is missing we don't want an empty value, as that would look very silly when viewed in client.